components/layout/__tests__/navigation.test.tsx

```diff
@@ -7,140 +7,13 @@ jest.mock("next/router", () => require("next-router-mock"));
-          href="https://bit.ly/hotlinewarga"
-          rel="nofollow noopener noreferrer"
-          target="_blank"
-        >
-          <svg
-            aria-hidden="true"
-            class="w-8 h-8"
-            fill="none"
-            stroke="currentColor"
-            viewBox="0 0 24 24"
-            xmlns="http://www.w3.org/2000/svg"
-          >
-            <path
-              d="M8 10h.01M12 10h.01M16 10h.01M9 16H5a2 2 0 01-2-2V6a2 2 0 012-2h14a2 2 0 012 2v8a2 2 0 01-2 2h-5l-5 5v-5z"
-              stroke-linecap="round"
-              stroke-linejoin="round"
-              stroke-width="2"
-            />
-          </svg>
-          <span
-            class="text-xs truncate"
-          >
-            Hubungi Kami
-          </span>
-        </a>
-      </li>
-      <li
-        class="relative"
-      >
-        <a
-          class="inline-flex flex-col items-center justify-center text-center h-12 px-2 rounded-md text-gray-600 hover:text-blue-600"
-          href="/provinces"
-        >
-          <svg
-            aria-hidden="true"
-            class="w-8 h-8"
-            fill="none"
-            stroke="currentColor"
-            viewBox="0 0 24 24"
-            xmlns="http://www.w3.org/2000/svg"
-          >
-            <path
-              d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
-              stroke-linecap="round"
-              stroke-linejoin="round"
-              stroke-width="2"
-            />
-          </svg>
-          <span
-            class="text-xs truncate"
-          >
-            Pusat Data
-          </span>
-        </a>
-      </li>
-      <li
-        class="relative"
-      >
-        <a
-          class="inline-flex flex-col items-center justify-center text-center h-12 px-2 rounded-md text-gray-600 hover:text-blue-600"
-          href="/faq"
-        >
-          <svg
-            aria-hidden="true"
-            class="w-8 h-8"
-            fill="none"
-            stroke="currentColor"
-            viewBox="0 0 24 24"
-            xmlns="http://www.w3.org/2000/svg"
-          >
-            <path
-              d="M4 6a2 2 0 012-2h2a2 2 0 012 2v2a2 2 0 01-2 2H6a2 2 0 01-2-2V6zM14 6a2 2 0 012-2h2a2 2 0 012 2v2a2 2 0 01-2 2h-2a2 2 0 01-2-2V6zM4 16a2 2 0 012-2h2a2 2 0 012 2v2a2 2 0 01-2 2H6a2 2 0 01-2-2v-2zM14 16a2 2 0 012-2h2a2 2 0 012 2v2a2 2 0 01-2 2h-2a2 2 0 01-2-2v-2z"
-              stroke-linecap="round"
-              stroke-linejoin="round"
-              stroke-width="2"
-            />
-          </svg>
-          <span
-            class="text-xs truncate"
-          >
-            Info Umum
-          </span>
-        </a>
-      </li>
-    </ul>
-  </div>
-</nav>
-`);
+    const homeText = screen.getByText("Beranda");
+    const anchorHome = homeText.closest("a");
+
+    expect(homeText).toBeInTheDocument();
+    expect(anchorHome).toHaveAttribute("href", "/");
```

Comment:
Please assert all the links shown in this navigation component, since it's the main feature and we need to assert that it's not broken at all.

You can do the same thing as I did for the `bottomNavigation` [over here](https://github.com/kawalcovid19/wargabantuwarga.com/blob/85f7b0e0992cafcff406ad37ee30ff5b40c3872f/components/layout/__tests__/navigation.test.tsx#L153).